bindings/swig/conf.i

```diff
@@ -15,10 +16,8 @@
     #include <iterator>
     #include "libdnf/conf/ConfigRepo.hpp"
     #include "libdnf/conf/ConfigParser.hpp"
-    using namespace libdnf;
```

Comment:
Why you remove it?

Reply:
In fact it was a fallback. Some symbols were not prefixed by namespace in C++ code generated by swig.
After my last changes it is not needed.